src/contract_wrappers/tokens/security_token_wrapper/__tests__/common.test.ts

```diff
@@ -2174,7 +2174,7 @@ describe('SecurityTokenCommon', () => {
         from: '0x1111111111111111111111111111111111111111',
         to: '0x2222222222222222222222222222222222222222',
         value: new BigNumber(1),
-        data: 'string',
+        data: stringToBytes32(''),
```

Comment:
Why doesn't this use a hex string with transfer data?

Reply:
It wouldn't help to test deeply this method. Since hex data is not processed at this layer, only at contract level.